app/src/main/java/de/westnordost/streetcomplete/location/LocationUtil.kt

```diff
@@ -0,0 +1,26 @@
+package de.westnordost.streetcomplete.location
+
+import android.Manifest
+import android.content.Context
+import android.content.IntentFilter
+import android.content.pm.PackageManager
+import android.location.Location
+import android.location.LocationManager
+
+import androidx.core.content.ContextCompat
+import androidx.core.location.LocationManagerCompat
+
+import de.westnordost.streetcomplete.data.osm.mapdata.LatLon
+
+fun isLocationEnabled(context: Context): Boolean =
+    hasLocationPermission(context) && LocationManagerCompat
+        .isLocationEnabled(ContextCompat.getSystemService(context, LocationManager::class.java)!!)
+
+fun hasLocationPermission(context: Context): Boolean =
+    ContextCompat.checkSelfPermission(context,
+        Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED
```

Comment:
I'm never sure how I want to format extremely long lines like this that Kotlin sometimes produces, so I just left them the same :P

Reply:
🤷 
Well I guess you could import `android.Manifest.permission.ACCESS_FINE_LOCATION` and `PERMISSION_GRANTED` and then just write 
```kotlin

ContextCompat.checkSelfPermission(context, ACCESS_FINE_LOCATION) == PERMISSION_GRANTED
```
You choice, I don't mind either way.

Reply:
Ok, that works.